marketplace-and-integrations/packages/authorized-services.md

```diff
@@ -44,13 +47,24 @@ Using a settings screen the administrator can review the service configuration.
 
 ![settings-screen](images/settings-screen.png)
 
+Depending on the authentication method of the service,
+- `OAuth1`
+- `OAuth2` (default)
+- `ApiKey`
+
+the interface provides two methods for retrieving the value of the access token or the API key:
+- `GetToken`
+- `GetApiKey` 
+
 ## Usage
 
 In the following, you can learn more about how to use the Umbraco Authorized Services package.
 
 ### App Creation
 
-Services that this package is intended to support will offer an OAuth authentication and authorization flow against an "app". The developer will need to create this "app" with the service.  By doing this, information such as the "client ID" and "client secret" can be applied to the configuration.
+Services that this package is intended to support will offer an OAuth2 default authentication and authorization flow against an "app". The developer will need to create this "app" with the service.  By doing this, information such as the "client ID" and "client secret" can be applied to the configuration.
```

Comment:
```suggestion
The primary use case for this package is when working with services that offer an OAuth2 authentication and authorization flow against an "app". The developer will need to create this "app" with the service.  By doing this, information such as the "client ID" and "client secret" can be applied to the configuration.
```

Reply:
We should also add here a section I've added to the readme:

```
When creating the app it will usually be necessary to configure a call back URL. You should use the following:

- For OAuth2: `/api/AuthorizedServiceResponse/HandleOAuth2IdentityResponse`
- For OAuth1: `/api/AuthorizedServiceResponse/HandleOAuth1IdentityResponse`
```

Reply:
Updated